File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.go
func sumOfUnique(nums []int) int {
    hashMap := make(map[int]int)
    
    for i := range nums {
        _, ok := hashMap[nums[i]]
        if ok {
            hashMap[nums[i]]++
        } else {
            hashMap[nums[i]] = 1
        }
    }
    
    answer := 0
    for k, v := range hashMap {
        if v == 1 {
            answer += k
        }
    }
    
    return answer
}

File: 1748-sum-of-unique-elements/1748-sum-of-unique-elements.java
import java.util.Map;
import java.util.HashMap;

class Solution {
    public int sumOfUnique(int[] nums) {
        HashMap<Integer,Integer> hashMap = new HashMap<>();
        for (int num : nums) {
            Integer count = hashMap.get(num);
            if (count == null) {
                hashMap.put(num, 1);
            } else {
                hashMap.put(num, count + 1);
            }
        }

        int answer = 0;
        for (Map.Entry<Integer,Integer> entry: hashMap.entrySet()) {
            if (entry.getValue().equals(1)) {
                answer += entry.getKey();
            }
        }
        return answer;
    }
}